packages/docker-engine/docker-engine.spec

```diff
@@ -7,8 +7,11 @@
 %global dorepo docker
 %global doimport %{goproject}/%{dorepo}
 
-%global gover 18.09.9
+%global gover 19.03.12
 %global rpmver %{gover}
+%global gitrev 9dc6525e6118a25fab2be322d1914740ea842495
+
+%global source_date_epoch 1363394400
```

Comment:
Where does this timestamp come from?

Reply:
This is, as near as I can tell, Docker's birthday.  (The date and time of [this talk](https://pyvideo.org/pycon-us-2013/the-future-of-linux-containers.html).)